routes.js

```diff
@@ -50,10 +50,11 @@ export default function getRoutes(
 	{
 		API_SERVER_ROOT_URL,
 		PHOTO_SCALER_SALT,
-	}) {
+	},
+	apiProxyFn$ = apiProxy$) {
```

Comment:
this is actually a big improvement prompted by writing tests IMO - we can make this function more 'pure' _and_ more testable by making this a function arg rather than consuming it from the module scope. By using ES6 default args, it's also not a breaking change, just more flexibility - in theory, you could use this module with any API proxying routine now.